AzureCalling/Controllers/CallViewController.swift

```diff
@@ -520,14 +521,44 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         meetingInfoViewUpdate()
     }
 
-    @objc func recordingActiveChangeUpdated(_ notification: Notification) {
-        guard let isRecordingActive = callingContext?.isRecordingActive else {
+    private func toggleMeetingCaptureNotification(recording: NSAttributedString, transcribing: NSAttributedString, bothActive: NSAttributedString, stopped: NSAttributedString) {
+        guard let isRecordingActive = callingContext?.isRecordingActive,
```

Comment:
I think we could combine the word logic into the toggleNotification function
You could have a look on Android side implement method (use an enum as input) or implement a simple `state machine`: https://albertodebortoli.com/2018/12/16/the-easiest-state-machine-in-swift/

Reply:
Could you please clarify the Android side implementation method using enums? I'm seeing basic implementation using [if-else logic](https://github.com/Azure-Samples/communication-services-android-calling-hero/blob/2d529525085f759fa02f16efa7431f742d5a0170/AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java#L378-L404).

Also wondering if we are to use state machines, could we just utilize the [Stateful](https://github.com/albertodebortoli/Stateful) package by including `pod 'Stateful'` in the podfile?

Reply:
I've changed the `toggleMeetingCaptureNotification` method to accept an enum. That way if there are more types added in the future, the switch statement grows instead of the parameter list of a function.

Personally I would prefer an actual state machine implementation, but for now that might be an overkill.